products/facilities/regional-offices/engineering/testing-qa.md

```diff
@@ -1 +1,90 @@
+# Testing
+## Overview of VBA Regional Benefit Offices product and complexity
+The product is a page generated from CMS editor content using templates in content-build with one component being brought in from vets-website. The page is made up of several templates from other products as well as some new pieces. Much of the beginning of the VBA Regional Benefit Office page is centralized content in a departure from VAMC, e.g. the intro text is centralized and uniformed across all facilities, leading to more certainty in testing. Shared content like the top task links (e.g. make an appointment), the On-this-page contents, operating hours display, Medallia survey button, back-to-top component etc. make the product less complex to test. 
```

Comment:
> several templates from other products

Does this mean like partials? Or like we used existing templates as a starting point / duped them?

Reply:
s/uniformed/uniform

Reply:
partials, but not sure if they know the term